src/main.rs

```diff
@@ -19,7 +19,7 @@ use rdkafka::Message;
 use avro_rs::{from_value};
 use schema_registry_converter::schema_registry_common::SubjectNameStrategy;
 use schema_registry_converter::blocking::avro::{AvroEncoder, AvroDecoder};
-use schema_registry_converter::blocking::schema_registry::SrSettings;
+use schema_registry_converter::blocking::schema_registry::{SrSettings};
```

Comment:
`rustfmt` foretrekker den slik. Kan kanskje også ta en ny PR på å formatere hele dette prosjektet.

```suggestion
use schema_registry_converter::blocking::schema_registry::SrSettings;
```